lib/driver/compilerTool.cpp

```diff
@@ -44,31 +56,53 @@ int Compiler([[maybe_unused]] struct DriverCompilerOptions &Opt) noexcept {
   if (Opt.PropBulkMemOps.value()) {
     Conf.removeProposal(Proposal::BulkMemoryOperations);
   }
-  if (Opt.PropRefTypes.value()) {
-    Conf.removeProposal(Proposal::ReferenceTypes);
-  }
   if (Opt.PropSIMD.value()) {
     Conf.removeProposal(Proposal::SIMD);
   }
-  if (Opt.PropRelaxedSIMD.value()) {
-    Conf.addProposal(Proposal::RelaxSIMD);
+  if (Opt.PropTailCall.value()) {
+    Conf.removeProposal(Proposal::TailCall);
+  }
+  if (Opt.PropExtendConst.value()) {
+    Conf.removeProposal(Proposal::ExtendedConst);
   }
   if (Opt.PropMultiMem.value()) {
-    Conf.addProposal(Proposal::MultiMemories);
+    Conf.removeProposal(Proposal::MultiMemories);
```

Comment:
It was originally `addProposal(Proposal::MultiMemories)`, is this change the purpose or accidentally made?